src/cmd/barrierd/barrierd.rc

```diff
@@ -1,6 +1,6 @@
 // Microsoft Visual C++ generated resource script.
 //
-#include "resource.h"
+//#include "resource.h"
```

Comment:
Any reason for commenting this out versus removing the line?

Reply:
No, not really... I can remove it, for sure.